src/main/java/org/prebid/server/bidder/rubicon/RubiconBidder.java

```diff
@@ -401,19 +412,26 @@ private Imp makeImp(Imp imp,
         final App app = bidRequest.getApp();
         final Site site = bidRequest.getSite();
         final ExtRequest extRequest = bidRequest.getExt();
+        final boolean isVideo = isVideo(imp);
+
+        final PriceFloorResult priceFloorResult = isVideo ? resolvePriceFloors(bidRequest, imp) : null;
```

Comment:
Seems if video is false, than it is possible banner, than floors function would be great to call with `banner` mediatype